sentry-core/src/main/java/io/sentry/core/PiiEventProcessor.java

```diff
@@ -0,0 +1,33 @@
+package io.sentry.core;
+
+import io.sentry.core.protocol.User;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Removes personal identifiable information from {@link SentryEvent} if {@link
+ * SentryOptions#isSendDefaultPii()} is set to false.
+ */
+@ApiStatus.Internal
```

Comment:
if the class is not public and only package-private, we don't need `@ApiStatus.Internal` I'd say.